include/EMaterialProps.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2002-2012 Nikolaus Gebhardt
+// This file is part of the "Irrlicht Engine".
+// For conditions of distribution and use, see copyright notice in irrlicht.h
+
+#ifndef __E_MATERIAL_PROPS_H_INCLUDED__
+#define __E_MATERIAL_PROPS_H_INCLUDED__

```

Comment:
fyi there's no harm in changing enum values as you please
(unless you renamed it for another reason)

Reply:
I renamed it because "flag" sounded like "boolean value" to me, but not all of them are boolean values. Wasn't really necessary though.